### DISCUSSION TITLE
Integration with homeassistant

### DISCUSSION THREAD

**Discussion by Carstenst** [fabian-lauer/dbus-shelly-3em-smartmeter#31] [Q&A]
Hi,
I have homeassitant running on a raspberry and tried to install it on that system. In the end the installation step does not work as the install-file can´t be found.
I´m not that experienced and just started my first steps. Maybe homeassistant is the wrong approach or I´ve made another mistake.

Carsten

**Comment by fabian-lauer** [comment]
Can you maybe hardcopy your screen when performing installation?!
Thx

**Comment by Carstenst** [comment]
![screenshot](https://user-images.githubusercontent.com/126786240/223507579-793d5265-f0d9-4d7c-b7a6-0a7519048f2a.JPG)

**Comment by fabian-lauer** [comment]
Problem was solved in parallel - please see https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/issues/32
Added changes to main branch - should work now.

**Comment by Carstenst** [comment]
many thanks! Will try it again...

**Comment by Carstenst** [comment]
sorry, there´s the same issue at the same point...

![screenshot 2](https://user-images.githubusercontent.com/126786240/223529255-2d68e246-b336-4a58-9790-3ea4ccba1c74.JPG)

**Comment by fabian-lauer** [comment]
Ah @Carstenst  now I got it a bit better - reading and checking helps :-D

On the venusOS devices a symlink is created in dir "/service" is doesn't exist on your end.....as mentioned only test on my CerboGX device.
For raspberry not sure :-(

You can try to uncomment / remove the line 18 from the install-script, but not sure if the script will start as deamon or automated on your end
